### ISSUE TITLE
Update Project Profile: LA TDM Calculator Remove Ebi Imafidon

### ISSUE THREAD

**Issue by theogguu** [hackforla/website#7650]
### Prerequisites
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started](https://www.hackforla.org/getting-started) page and attend an onboarding session.
2. You have already read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
We need to keep project information up to date so that visitors to the website can find accurate information.

### Action Items
- [x] In your IDE, open the `_projects/tdm-calculator.md` file.
- [x] Observe the existing syntax of the front matter block [^1] in the file.
- [x]  Find the `leadership` variable and remove the following:
```
  - name: Ebi Imafidon
    github-handle:
    role: Product Manager
    links:
      slack: 'https://hackforla.slack.com/team/U031EU2E6LB'
      github: 'https://github.com/Biuwa'
    picture: https://avatars.githubusercontent.com/Biuwa
```
- [x] Verify the changes by viewing the following in your local environment and include before and after screenshots with your pull request:
  - [x] LA TDM Calculator page [^2]
- [ ] Once your pull request is merged, go to the initiating ER [^3]
  - [ ] Check off this issue under the _Dependency_ section
  - [ ] If all the issues in the _Dependency_ section are checked off, move initiating ER [^3] to **Questions / In Review** column and uncheck the **Dependency label**.
    - <details>
        <summary>Click here to see how to uncheck the <b>Dependency label</b></summary>
        <img src="https://github.com/hackforla/website/assets/31293603/6f53f4d4-7d2c-45f8-8534-9936fc9adee8" width="300px">
      </details>

### Resources/Instructions
[^1]: [Info about the front matter block](https://jekyllrb.com/docs/front-matter/)
[^2]: Project detailed info page URL: https://www.hackforla.org/projects/tdm-calculator
[^3]: initiating ER:  #7305

**Comment by HackforLABot** [comment]
Hi @mugdhchauhan, thank you for taking up this issue! Hfla appreciates you :)

Do let fellow developers know about your:-
i. **Availability:** (When are you available to work on the issue/answer questions other programmers might have about your issue?)
ii. **ETA:** (When do you expect this issue to be completed?)

You're awesome!

P.S. - You may not take up another issue until this issue gets merged (or closed). Thanks again :)

**Comment by mugdhchauhan** [comment]
Availability for this week:

* Daily @ 6 PM - 8 PM (GMT)

My estimated ETA for completing this issue:

* 22 Nov 2024 @ 7 PM (GMT)

**Comment by mugdhchauhan** [comment]
**Progress**: _"What is the current status of your project? What have you completed and what is left to do?"_

- [X] Synced my fork and deleted my branch that had been merged `correct-misspelling-about-page-7623`
- [X] Created a new branch `LA-TDM-Calculator-remove-Ebi-Imafidon-7650`
- [X] In my IDE, opened the `_projects/tdm-calculator.md` file
- [X] Observed the existing syntax of the front matter block in the file
- [X] Found the leadership variable and removed the information for Ebi Imafidon
- [X] Verified the changes by viewing in my local environment

**Blockers:** _"Difficulties or errors encountered."_

- No Blockers

**Availability:** _"How much time will you have this week to work on this issue?"_

- I am available daily @ 6 PM - 8 PM (GMT)

**ETA:** _"When do you expect this issue to be completed?"_

- Completed updating project profile by removing Ebi Imafidon

**Pictures (optional):** _"Add any pictures of the visual changes made to the site so far."_

- Screenshot before changes
![Screenshot Before](https://github.com/user-attachments/assets/9d1d3eff-0e91-4990-8fa9-eb8d87388bc7)

- Screenshot after changes
![Screenshot After](https://github.com/user-attachments/assets/ff6fc8d2-1a30-4a0b-bc01-0920654c2cbf)
